samples/AspNetCore/RoutingSample/Startup.cs

```diff
@@ -1,150 +1,161 @@
-                    return;
-                }
-
-                account.Balance -= transaction.Amount;
-                await client.SaveStateAsync(StoreName, transaction.Id, account);
-
-                context.Response.ContentType = "application/json";
-                await JsonSerializer.SerializeAsync(context.Response.Body, account, serializerOptions);
-            }
-        }
-    }
-}
+﻿// ------------------------------------------------------------
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+// ------------------------------------------------------------
+
+namespace RoutingSample
+{
+    using System;
+    using System.IO;
+    using System.Text;
+    using System.Text.Json;
+    using System.Threading.Tasks;
+    using Dapr;
+    using Dapr.Client;
+    using Microsoft.AspNetCore.Builder;
+    using Microsoft.AspNetCore.Hosting;
+    using Microsoft.AspNetCore.Http;
+    using Microsoft.Extensions.Configuration;
+    using Microsoft.Extensions.DependencyInjection;
+    using Microsoft.Extensions.Hosting;
+
+    /// <summary>
+    /// Startup class.
+    /// </summary>
+    public class Startup
+    {
+        /// <summary>
+        /// State store name.
+        /// </summary>
+        public const string StoreName = "statestore";
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="Startup"/> class.
+        /// </summary>
+        /// <param name="configuration">Configuration.</param>
+        public Startup(IConfiguration configuration)
+        {
+            this.Configuration = configuration;
+        }
+
+        /// <summary>
+        /// Gets the configuration.
+        /// </summary>
+        public IConfiguration Configuration { get; }
+
+        /// <summary>
+        /// Configures Services.
+        /// </summary>
+        /// <param name="services">Service Collection.</param>
+        public void ConfigureServices(IServiceCollection services)
+        {
+            services.AddDaprClient();
+
+            services.AddSingleton(new JsonSerializerOptions()
+            {
+                PropertyNamingPolicy = JsonNamingPolicy.CamelCase,
+                PropertyNameCaseInsensitive = true
+            });
+
+            services.AddControllers();
+        }
+
+        /// <summary>
+        /// Configures Application Builder and WebHost environment.
+        /// </summary>
+        /// <param name="app">Application builder.</param>
+        /// <param name="env">Webhost environment.</param>
+        /// <param name="serializerOptions">Options for JSON serialization.</param>
+        public void Configure(IApplicationBuilder app, IWebHostEnvironment env, JsonSerializerOptions serializerOptions)
+        {
+            if (env.IsDevelopment())
+            {
+                app.UseDeveloperExceptionPage();
+            }
+
+            app.UseRouting();
+            
+            app.UseCloudEvents();
+            app.UseTwilioWebHooks();

```

Comment:
Routing sample is specific to showing Dapr pubsub and state AspNetCore integration, it should not use TwillioWebHooks

Reply:
Ok, but it also is used to work with cloud events, and this is another example of middleware... should I create another sample application to show its use?

Reply:
This middleware is not invoked when Actor POST URL is used... I would really like to have this middleware do the same thing as it does when posting to the -app-port vs the --port